pkgs/development/python-modules/pulsectl-asyncio/default.nix

```diff
@@ -0,0 +1,20 @@
+{ buildPythonPackage, fetchPypi, lib, pulsectl }:
+
+buildPythonPackage rec {
+  pname = "pulsectl-asyncio";
+  version = "0.2.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-qBMIH/HOnt9SXrnrGtZTG4fOb0fftR7mBaU7K4mcYvo=";
+  };
+
+  propagatedBuildInputs = [ pulsectl ];
+
```

Comment:
You should add
```
checkInputs = [
    pulsectl-asycio
];
```
Or whatever the library is supposed to be imported as! This will test if the library can at least be imported

If there are any other unit tests available (which there seems to be) you should look at how to add them as well!